apps/laboratory/package.json

```diff
@@ -46,9 +46,9 @@
     "react-icons": "4.12.0",
     "@walletconnect/utils": "2.12.0",
     "valtio": "1.11.2",
-    "viem": "2.10.5",
+    "viem": "2.10.3",
     "wagmi": "2.8.7",
-    "@wagmi/core": "2.9.7"
+    "@wagmi/connectors": "4.3.9"
```

Comment:
connectors is to force the proper version of the provider, correct?
but why remove core?

Reply:
added the core dependency as it is being used.